### ISSUE TITLE
use android_id: ^0.1.3+1 

### ISSUE THREAD

**Issue by vicentxhc** [fluttercommunity/android_id#30]
use android_id: ^0.1.3+1 release open app crash
log:
```
Flutter run key commands.
h List all available interactive commands.
c Clear the screen
q Quit (terminate the application on the device).
W/FlutterActivityAndFragmentDelegate(13131): A splash screen was provided to Flutter, but this is deprecated. See flutter.dev/go/android-splash-migration for migration steps.
E/AndroidRuntime(13131): FATAL EXCEPTION: pool-2-thread-1
E/AndroidRuntime(13131): Process: mo.edu.iftm.app, PID: 13131
E/AndroidRuntime(13131): java.lang.IllegalAccessError: Illegal class access: 'd.c.a.a.i.I.c' attempting to access 'd.c.a.a.i.z' (declaration of 'd.c.a.a.i.I.c' appears in /data/app/~~TvkTPrSul9oPWUlIK-jGiA==/mo.edu.iftm.app-3H9t9GfBIEGRQAoO20q-nQ==/base.apk)
E/AndroidRuntime(13131):        at d.c.a.a.i.I.c.c(:2)
E/AndroidRuntime(13131):        at d.c.a.a.i.I.a.run(Unknown Source:8)
E/AndroidRuntime(13131):        at d.c.a.a.i.s.run(Unknown Source:2)
E/AndroidRuntime(13131):        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/AndroidRuntime(13131):        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/AndroidRuntime(13131):        at java.lang.Thread.run(Thread.java:920)
```

**Comment by nohli** [comment]
Can you please format the log?

What happens if you remove android_id from pubspec.yaml, does it still crash?

**Comment by vicentxhc** [comment]
> Can you please format the log?
> 
> What happens if you remove android_id from pubspec.yaml, does it still crash?

remove android_id no crash，why?

**Comment by nohli** [comment]
Can you try deleting the `android` folder, and then recreate it via `flutter create .`?

**Comment by nohli** [comment]
Can you please try `android_id: ^0.2.0`

**Comment by xalikoutis** [comment]
when app is released crashes with android_id: ^0.3.2

**Comment by nohli** [comment]
@xalikoutis it doesn't crash in debug mode? How can I reproduce the crash?
